model/src/main/proto/performance_metrics.proto

```diff
@@ -29,7 +29,7 @@ message OppiaMetricLog {
   LoggableMetric loggable_metric = 7;
 
   // The current screen where the event is getting logged.
-  CurrentScreen current_screen = 8;
+  string current_screen = 8;
```

Comment:
Could we use ScreenName or CurrentAppScreen here, instead?

Reply:
Done.